### PR TITLE
Add a new check for bugs tracked with P4 or P5 priority

### PR DESCRIPTION
**Pull request by sylvestre**

### CODE REVIEW

File: auto_nag/scripts/mismatch-priority-tracking.py [hunk 1]
```diff
@@ -0,0 +1,57 @@
...
+
+        release_version, beta_version, central_version = getVersions()
+        params = {'bug_status': ['---', 'FIXED',
+                                 'INVALID', 'WONTFIX',
+                                 'DUPLICATE', 'WORKSFORME',
+                                 'INCOMPLETE', 'SUPPORT',
+                                 'EXPIRED', 'MOVED'],
+                  'j_top': 'OR',
+                  'priority': ['p4', 'p5'],
+                  'o2': 'anywordssubstr',
```
Comment (calixteman) [inline auto_nag/scripts/mismatch-priority-tracking.py]: 'o2' is used two times

File: auto_nag/scripts/mismatch-priority-tracking.py [hunk 2]
```diff
@@ -0,0 +1,57 @@
...
+                  'short_desc_type': 'allwordssubstr',
```
Comment (calixteman) [inline auto_nag/scripts/mismatch-priority-tracking.py]: what's the goal of this field ?

File: auto_nag/scripts/mismatch-priority-tracking.py (hunk 1 above)
Comment (sylvestre) [reply auto_nag/scripts/mismatch-priority-tracking.py]: already fixed :)
refresh

File: auto_nag/scripts/mismatch-priority-tracking.py (hunk 2 above)
Comment (sylvestre) [reply auto_nag/scripts/mismatch-priority-tracking.py]: dunno, it was in the query, removing it.

File: auto_nag/scripts/mismatch-priority-tracking.py [hunk 3]
```diff
@@ -0,0 +1,57 @@
...
+
+    def get_bz_params(self, date):
+
+        release_version, beta_version, central_version = getVersions()
+        params = {'bug_status': ['---', 'FIXED',
+                                 'INVALID', 'WONTFIX',
+                                 'DUPLICATE', 'WORKSFORME',
+                                 'INCOMPLETE', 'SUPPORT',
+                                 'EXPIRED', 'MOVED'],
+                  'j_top': 'OR',
```
Comment (calixteman) [inline auto_nag/scripts/mismatch-priority-tracking.py]: I don't know how OR is applyed here:
priority=[...] OR cf_tracking... OR cf_tracking...
And I suppose you want priority=... AND (cf_tracking... OR cf_tracking ...)

**Pull request merged**